CMakeLists.txt

```diff
@@ -389,13 +389,13 @@ if(NOT OPUS_DISABLE_INTRINSICS)
     if(OPUS_X86_MAY_HAVE_SSE)
       add_sources_group(opus celt ${celt_sources_sse})
       target_compile_definitions(opus PRIVATE OPUS_X86_MAY_HAVE_SSE)
-      if(NOT MSVC)
+      if(NOT (MSVC AND CMAKE_C_COMPILER_ID STREQUAL "MSVC"))
```

Comment:
Maybe easier to make a variable for the case when using MSVC compiler frontend and not.

Reply:
Good point! Updated.